catalogue/webapp/components/StoriesGrid/StoriesGrid.tsx

```diff
@@ -161,18 +161,21 @@ const StoryInformation = styled(Space).attrs({
 `;
 
 const StoryInformationItem = styled.span`
-  &:not(:first-child)::before {
-    content: ' | ';
-    margin: 0 4px;
-  }
-
   span {
     &:not(:first-child)::before {
       content: ', ';
     }
   }
 `;
 
+const StoryInformationItemSeparator = styled.span`
```

Comment:
This is a very cool fix! Maybe leave a comment for future joiners to explain how this works and why we want it. 💯